recipes/yaqd-core/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "yaqd-core" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "2d40037fb0506294cc82f879ffe0baeb7ed1c3c96572f6587e2e920ecd261fc3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

But you need the ``entry_points`` here
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
  entry_points:
    - yaqd-base=yaqd_core._daemon:Base.main
    - yaqd-hardware=yaqd_core._hardware._base:Hardware.main
    - yaqd-continuous-hardware=yaqd_core._hardware._continuous:ContinuousHardware.main
```

Thanks to [grayskull](https://github.com/marcelotrevisani/grayskull)